examples/message-ubench/main.pony

```diff
@@ -132,9 +138,29 @@ actor SyncLeader
     _env = env
     _initial_pings = initial_pings
     _ps = ps'
+
+    // Create timers
+    _timers = Timers
+
+    // Create the interval timer which will be cause the pinging
+    let interval: U64 = (report_interval.u64() * 1_000_000_000) / 10
+    let interval_timer = Timer(Tick(env, this), interval, interval)
+    _interval_timer = interval_timer
```

Comment:
this isn't needed.

Reply:
I couldn't get that to work.

Reply:
It should work. Please switch and if there is a problem, I can see the error in CI and tell you how to fix.

Reply:
I'll work on more.